bundles/python/python_lexer.moon

```diff
@@ -46,12 +46,15 @@ howl.aux.lpeg_lexer ->
   operator = c 'operator', S'+-*/%~&^=!<>;:,.(){}[]|`'
 
   name = (alpha + '_')^1 * (alpha + digit + P'_')^0
+  user_defined_constant = c('constant', (upper + '_')^1) * -#(alpha + digit)
```

Comment:
Ah - good point. Fixed to exclude single underscore. Will still highlight constants starting with underscore (`_EXAMPLE`).